RefrigeratorExe/RefrigeratorExe/Refrigerator.cs

```diff
@@ -0,0 +1,185 @@
+        public int NumberOfShelfs { get; set; }
+        public List<Shelf> Shelves;
+
+
+        public Refrigerator(string model, string color, int shelf)
+        {
+            id = Guid.NewGuid();
+            try
+            {
+                Model = model;
+
+            }
+            catch (Exception e)
+            {
+
+                Console.WriteLine(e.Message); 
+            }
+            Color = color;
+            NumberOfShelfs = shelf;
+            Shelves = new List<Shelf>();
+            this.AddShelfs();
+        }
+        public void AddShelfs()
+        {
+            while (this.Shelves.Count() < this.NumberOfShelfs)
+            {
+                this.Shelves.Add(new Shelf(this.Shelves.Count()));
+            }
+        }
+        public override string ToString()
+        {
+            string shelfs = "";
+            foreach (Shelf shelf in Shelves)
+            {
+                shelfs += "\n=================================================================================\n";
+                shelfs += "\n" + shelf.ToString();
+            }
+            return "Refrigerator id: " + id + " Model: " + Model + " Color: " + Color + " with " + NumberOfShelfs + " shefls containes \n" + shelfs;
+        }
+        public int SpaceInRefrigerator()
+        {
+            int spaceSum = 0;
+            foreach (Shelf shelf in Shelves)
+                spaceSum += shelf.Space;
+            return spaceSum;
+        }
+        public bool AddItem(Item item)
+        {
+            int count = 0;
+            foreach (Shelf shelf in Shelves)
+            {
+                if (shelf.IsSpaceInShelf(item.Space))
+                {
+                    shelf.AddItem(item);
+                    return true;
+                }
+                else
+                {
+                    if (item.Space > shelf.ShelfSpace)
+                        count++;
+                }
+
+            }
+            if (count == this.Shelves.Count)
+                Console.WriteLine("There is no shelf with this amount of space this item is too big");
+            else
+                Console.WriteLine("There is no more space in Refrigeratir, you might want to clean refrigerator!");
+            return false;
+        }
+        public Item GetItem(string id)
+        {
+            Item item;
+            foreach (Shelf shelf in Shelves)
+            {
+                item = shelf.InShelf(id);
+                if (item != null)
+                {
+                    shelf.RemoveItem(item);
+                    return item;
+                }
+            }
+            Console.WriteLine("There is no item with this id ");
+            return null;
+        }
+        public List<Item> GetItems()
+        {
+            List<Item> items = new List<Item>();
+            foreach(Shelf shelf in Shelves)
+                items.AddRange(shelf.Items);
+            return items;
+        }
+        public void CleanRefrigeraot()
```

Comment:
1. typo
2. as this is a method of Refrigirator you can simply call it Clean:

public void Clean()
{
    foreach (var shelf in Shelves)
    {
        shelf.CleanShelf();
    }
}